### PR TITLE
Update footer copyright year

### PR DESCRIPTION
**Pull request by adamklepacz**

### CODE REVIEW

File: apps/polskifrontend-front/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Footer extends React.Component {
   render() {
     return (
       <div className={styles.footer}>
-        <p>Copyright@2017 - <a className={styles['footer__link']} target="_blank" href="https://nafrontendzie.pl">Na Frontendzie</a></p>
+        <p>Copyright@2020 - <a className={styles['footer__link']} target="_blank" href="https://nafrontendzie.pl">Na Frontendzie</a></p>
```
Comment (poulch) [inline apps/polskifrontend-front/src/components/Footer/Footer.js:9]: `new Date().getFullYear()` było by bardziej elastyczne ;)

**Comment by mmiszy** [comment]
@all-contributors add @adamklepacz for code

**Comment by mmiszy** [comment]
@all-contributors add @poulch for review

**Comment by mmiszy** [comment]
@all-contributors add @adamklepacz for code

**Comment by mmiszy** [comment]
@all-contributors add @poulch for review

**Pull request closed without merging**